nest/06.glossary.md

```diff
@@ -0,0 +1,153 @@
+# 6 Glossary
+
+## A
+
+## B
+
+## C
+* Cache
+
+In Nest, the cache is a local copy of the list of available packages from Raven-OS server.
+The cache has to be kept up-to-date in order to install the latest versions of the packages.
+
+* Category
+
+Inside each repository, there are several categories.
+The packages are split between the repositories and their categories.
+Each package name must be unique in a category of a repository, but there can be several packages with the same name spread in several categories.
+
+## D
+* Dependency
+
+There is a dependency when a package depends on another one to function properly.
+
+* Dependency graph/Dependency tree
+
+The dependency graph is a directed graph representing the dependencies of packages.
+
+## E
+
+## F
+* `finest`
+
+`finest` is the binary of Nest targeted towards experienced users.
+
+* `finest group create <name> [--parent <group>]`
+
+The command of `finest` to create a group, with `@root` as a parent by default.
+
+* `finest group list`
+
+The command of `finest` to list all groups on the OS.
+
+* `finest group remove <name> [--parent <group>]`
+
+The command of `finest` to remove a group, with `@root` as a parent by default.
+
+* `finest merge`
+
+The command of `finest` to merge the dependency graph `finest` altered with the transactions you applied, with the dependency graph currently in use by the OS.
+
+* `finest requirement add <requirement> [--parent <group>]`
+
+The comand of `finest` to add a requirement to a group, by default `@root`.
+
+* `finest requirement remove <requirement> [--parent <group>]`
+
+The command of `finest` to remove a requirement from a group, by default `@root`.
+
+## G
+* Groups
+
+Groups are a way to put packages together, to handle their installation and uninstallation in a more efficient manner.
```

Comment:
```suggestion
Groups are a way to group packages together, in order to install and uninstall them in a more efficient manner.
```
(I know you used "put" to avoid saying "group" but you are defining groups *in the context of nest*, so referring to grouping things is not a problem.)